### PR TITLE
Update guides

### LINKED ISSUE #179: [PE-D] Inconsistent use of full stops
In the `Quick Start` section, some instruction step ends with a full stop while some does not.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: deyixtan/ped#4

### LINKED ISSUE #190: [PE-D] Invalid pool command example in Command summary section of the UG
Example given doesn't match with the format specified.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: deyixtan/ped#15

### PR DESCRIPTION
**Pull request by lyueyang**
Resolves #179

### CODE REVIEW

**Review by chewterence — COMMENTED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -286,10 +286,10 @@ Action | Format, Examples
 **add** | `add n/NAME p/PHONE a/ADDRESS d/TRIPDAY t/TRIPTIME [tag/TAG tag/TAG ...]` <br> e.g., `add n/Jenny Talia p/91234567 a/Yishun Avenue 4 d/FRIDAY t/1800 tag/female`
```
Comment (markmcwong) [inline docs/UserGuide.md]: should be `e.g.` instead of `e.g.,`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -286,10 +286,10 @@ Action | Format, Examples
...
 **list** | `list` 
 **edit** | `edit INDEX [n/NAME] [p/PHONE] [a/ADDRESS] [d/DAY] [t/TIME] [tag/TAG tag/TAG ...]` <br> e.g., `edit 8 a/Changi Airport d/SATURDAY`
```
Comment (markmcwong) [inline docs/UserGuide.md]: Same for this and the rows below

**Review by markmcwong — COMMENTED** [review]
LGTM, suggested some minor changes

**Pull request merged**